packages/app/src/overview/components/backup-set-secret-key-modal-dialog/backup-set-secret-key-modal-dialog.component.tsx

```diff
@@ -118,14 +125,27 @@ export const BackupSetSecretKeyModal: FunctionComponent<BackupSetSecretKeyModalP
           message={messages.backupSetSecretKeyModalDescription}
         />
         <Form onSubmit={handleSubmitClick}>
```

Comment:
🟢
> You shouldn't be able to continue without entering your password

[source](https://appnroll.slack.com/archives/C020D6DUC1W/p1636464523019800?thread_ts=1636462775.015400&cid=C020D6DUC1W)

Could you please handle this requirement as part of this PR? 🙏 It's my bad that I didn't update the story after an Aleksandra Rogucka comment -> I'm sorry 😞

Reply:
https://github.com/mudita/mudita-center/pull/518/commits/7c4332fa0bf00026c792a0ff5d8dd2ef3eed2aa5